File: src/data/Model/Cms360/Registration/ProgramCourseLink.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
    [Table("ProgramCourseLink", Schema = "Registration")]
    public partial class RegistrationProgramCourseLink
    {
        [Key]
        [Required]
        public Guid ProgramCourseLinkId { get; set; }
        [Required]
        public Guid ProgramDetailId { get; set; }
        [Required]
        public Guid ClassId { get; set; }
        [Required]
        public Guid CourseId { get; set; }
        [Required]
        public int StatusId { get; set; }
        [Required]
        public Guid LoggerId { get; set; }

    }


    [Table("VW_ProgramLinkVM", Schema = "Registration")]
    public partial class RegistrationProgramCourseLinkVM
    {
        [Key]
        [Required]
        public Guid ProgramCourseLinkId { get; set; }
        [Required]
        public Guid ProgramDetailId { get; set; }
        [Required]
        public Guid ClassId { get; set; }
        [Required]
        public Guid CourseId { get; set; }
        [Required]
        public int StatusId { get; set; }
        [Required]
        public String CourseName { get; set; }
        [Required]
        public String ClassName { get; set; }

        [Required]
        public String Description { get; set; }
        [Required]
        public Guid LoggerId { get; set; }

    }
}

File: src/ui/app/root/application/models/Registration/ProgramCourseLink.ts
export interface IRegistrationProgramCourseLink {
	programCourseLinkId : string;
	programDetailId : string;
	classId : string;
	courseId : string;
	statusId : number;
	loggerId : string;

}

export  interface RegistrationProgramCourseLinkVM {
    programCourseLinkId: string;
    programDetailId: string;
    classId: string;
    courseId: string;
    statusId: number;
    courseName: string;
    className: String;
    description: String;
    loggerId: string;
}

export  interface Examcourselist {
    
    examScheduleId: string;
    examTypeId: string;
    courseId: string;
    fullName: string;
    sectionCourseLinkId: string;
    programCourseLinkId: string;
    programDetailId: string;
    classId: string;
    totalMarks:number;
    examDate:Date;
    campusProgramId :string;
    failMasterId:string;
    gradingMasterId :string;
    statusId :number;
    month:string;
}